Index: client/perffarm-client.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- client/perffarm-client.py	(revision 50bb4b2443dd8ea20a946270601f951262cd3130)
+++ client/perffarm-client.py	(date 1581766197669)
@@ -19,6 +19,9 @@
 
 from settings_local import *
 from settings import *
+
+from client.settings import REPOSITORY_PATH
+
 API_URL = 'http://127.0.0.1:8000/'
 MACHINE_SECRET = '610f79063e62e6ad09460ac2c4e66da0386dc89b'
 if __name__ == '__main__':
Index: rest_api/rest_api/settings_local.py.in
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- rest_api/rest_api/settings_local.py.in	(revision 50bb4b2443dd8ea20a946270601f951262cd3130)
+++ rest_api/rest_api/settings_local.py.in	(date 1581729121116)
@@ -4,7 +4,7 @@
         'default': {
                 'ENGINE': 'django.db.backends.postgresql_psycopg2',
                 'NAME': 'pgperffarm',
-                'USER': 'chenzhang',
+                'USER': 'postgres',
                 'PASSWORD': 'password',
                 #'HOST': '/var/run/postgresql'
                 'HOST': '/tmp'
Index: client/settings.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- client/settings.py	(revision 50bb4b2443dd8ea20a946270601f951262cd3130)
+++ client/settings.py	(date 1581765954244)
@@ -1,12 +1,12 @@
 import os
 import sys
 
-# global configuration
-GIT_URL = 'https://github.com/postgres/postgres.git'
-REPOSITORY_PATH = '/tmp/git-postgres'
-BUILD_PATH = '/Users/chenzhang/anaconda3'
+# global configuration/Users/chenzhang
+GIT_URL = 'https://gitee.com/purpleyu/postgres.git'
+REPOSITORY_PATH = '$HOME/git-postgres/'
+BUILD_PATH = '$HOME/anaconda3'
 BIN_PATH = os.path.join(BUILD_PATH, 'bin')
-DATADIR_PATH = '/tmp/data-postgres'
+DATADIR_PATH = '$HOME/data-postgres'
 
 POSTGRES_CONFIG = {
     'shared_buffers': '1GB',
Index: client/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- client/.gitignore	(revision 50bb4b2443dd8ea20a946270601f951262cd3130)
+++ client/.gitignore	(date 1581765742995)
@@ -1,2 +1,3 @@
 .lock
 settings_local.py
+settings.py
\ No newline at end of file
